doc/GLCM.ipynb

```diff
@@ -0,0 +1,414 @@
5/9rJxxxhmyaNEiERHp6uoSEZGWlpYRsS0tLYd/djQ33XST1NTUHP43eTL+UyRCCBkrHK/8KMIcSQgZXzA/EkLGAy/5JvrKK6+U9evXy89+9jP1M8sa+Xfznuepsr9wzTXXSDKZPPyvo6PjpTaJEEJGBccrP4owRxJCxhfMj4SQ8cAxfSf6L3z605+W++67Tx5//HGZNGnS4fLW1lYROfSOYlvbsHSjp6dHvbv4F0KhkIRCWkjgWSNFRq5BUIW+mC9AICYi4oR0HT7TF8iBRcmHv+8PCWRwuQPcANGDuG8WEpaFcWw5qssjvVje4NNOCLHAF/tF8Ljn6w2yCRBbwg4ZKIoLJnEb/DldnmvUbUCSORPRLtM4onYZ5scBYjGTFAxUYWHPg8Q6dX+RFEwEi9tM67SonTmSA3tCRCTSq9tQjhjaAHwigRxOLcFBXVYGwicREd+AFiNZJTzAXmxkrOe8ek/vO575UcScI4/mke+vgOWBs9KqrL5tEMYmUnoDuGk8Px3v0vmlKoMtIw0XH1Bl+Se0bExEZCDToMrsZpwbWhdq6VPo3+pg7L7zdVndg8AgKSJD0/Vav/q838PYb/7uTaqsZiNORqlpuqycwAKVYFQnyZq39sPYA321qqz5zQdVWf9DE+DvF5bqC5a1G89ly+/0Wgz9g34tEREPXEcPPK+FXCIiwRR4rdZBGBv/mh6zPRfWg0iRiTt0zmi/B5t+IiBH3vtWfaMnInJB6jOqbGgmDJXF79yhygI5PO87d7Wqsmc78KerNhCZwTORiFTHh4VwzrEcZo4zr1R+DA254gsO56jiA7Xw92t9eo0m5xjOThl9XSlMxDKq0IDOm2iNi2BZWL4Ot6H+Rd2GZA2ut1yr136+EeemeIfOsUh4JiJStV6Pd64V5+jqPbq8d6nhLJHU9Vb34XrTb9LXNXs/PnCWY0ByNU+Pb+sjePMMTdNlA/PxNdkGWys8AEPFKut2ZTx8va3fqqVrgzOw0LFvuT5cFgyiYT8QgBUNclqEEzZIyJrA+djBsUjC58/i1yvNAFLJDjwO0U5wT9KuzxciIkdebh0gGDZxTCdNz/PkyiuvlHvuuUceeeQRmT59+oifT58+XVpbW+Whhx46XFYsFmX16tVy+umnH8tLEULImIL5kRBCMMyPhJDxxjF9En3FFVfIXXfdJffee6/E4/HD31OpqamRSCQilmXJ1VdfLTfeeKPMnj1bZs+eLTfeeKNEo1G59NJLX5YOEELIaID5kRBCMMyPhJDxxjHdRN92220iIrJq1aoR5T/84Q/lwx/+sIiIfO5zn5NcLieXX365JBIJWb58uTz44IPH/Iw/QggZSzA/EkIIhvmREDLeOKabaM/wXeMjsSxLrr/+ern++utfapsIIWTMwfxICCEY5kdCyHjj1bPvEEIIIYQQQgghY4yXZOd+JbDcQ//+QmgImwpzwDiHjKAiImUgYjUZh02GSxgLRtECBmwRbDtEZkYRbDIOZCs3WFt4yKQAzHvBIRwb7dGVFGrwey9ofAv1uL3lGm0PDPXgQY8C8SuyaDsR/FqWCwzjwNYoIhIc0rFlg6jPr2WJkmvBgx7tBDZxw+7LNYMXNMwlMq1nJv3td/z/Vr1ozFB/RUQ8EFuK4jUSSCFDvqERIW2pNH6a4RxdxzFs4DHK9Pmd4o8NG0I7D2CDb9Vj2lgavBhrYqNhvaC8Pdh4WmrRsfXfxcms/S1ae+ybh/WbTkrbYwP1ePE9ecI9qmzuqk/B2MWn7VRl62LY0oy481Zt4RYRqQZLPW3Yg9W7ddnEszth7P4f6bb11+Hx/eLHfq7K/u+33qXKLvn4avj7v/nBSlU2tBBbh2f8ozZNb/rpAhg7NBvs7ZDh6Rmn6jXZv6EJx/4zsJRvwYbWbJPOBWe+/zkY++DOeaps6aOXw9hq8JSKODARi4hs6NTjM3QCPiSsOmGrKvvhlCdg7NWdp6iyxw9gRbjv18P2cqtoSObjiL6TROwjlkTE8ESOaJees0g3vn7lJoJzywFsuy6AhwRU7cPrIzVVlwUH8DUsNVXXEd8DQyUzSR8y0jPxEy7SC3Tf2h7AfUvO0ONTsx23oftMnQOCvbhv9QsGVVkoia372ef1n/gHcAqQSY/q9b77IzquFMPGbXS2nPgYvn7tfI8+CLc9jce8b7E+47gBvEZ2vUvHWiUc60vp8Q0DA7aISAmchU1W9tRMvUasEq7XnaOf+BB/BJ8lIgO63kyz4X5ggx7f7AR8hkRPAUJPMhIZeR/mHMOTfvhJNCGEEEIIIYQQUiG8iSaEEEIIIYQQQiqEN9GEEEIIIYQQQkiF8CaaEEIIIYQQQgipkFErFnOClkjQYHQ6AvQF+KJ22IgIFkgU44YvxQMPgIOdA2Lr78SLYxAcIDlT2RCLhGVlv2FM0PfqDW+RoDErh3G9mVYQbGiCaXwQ0XbdOcvgjCqBR0S64LWMY54GsQYJWS6sywMpg0gNCBl8eTw4SLBmktch+ZZtENXlG3W99lQtdBARcdu11CGQwe1FcxHux2NWrAISMiC6ExEpR3WnTRIyu6Ttcf5UAcZa2aPKK3icyljn45OekGh8eDxv3v9+GDfrCi0qemrTbBgb2aeNGj7TOt2uN2HviTj2c2ffq8puefH1MDbcqdsQbsUitFk/1RIxyyAF2fKI7nPLNiwk6Vqpy4cMDjJnkk7q9kGcjGr26vW76U+44mALkBwahIhffVFLz/wgb84Jd8HfT5+SU2WBALiwicizD89XZV4bblcAiBpRfhMRiczRYzPgwyI1++cNqmzWh/fB2D35KarswV1zYeyFczaosl//aRmMTQIRU6wdbxYkkZz6SzwOe36l5WaLFy2Gsei6gOZSRKTmiO3q/O2j1ZjnaDktkpGKiPSdrQcxugUfZvxD+lpVaMb7JL5dT3pyDm6DG9X5xt+H11Jmqn49u4SP8vUbdb39i3G9VR26PD2x8oViyo8WWGz+LK63+D9aJJjBW1Wy0
```

Comment:
I would make this your summary line at the end of the notebook, and explicitly say something like:
```
we can distinguish cells that are in the assembly stage of the viral life cycle and are producing viral particles (crop 3) and those that are not (crops 1, 2 and 5)
```

but updating to call them "cell 1, 2, 3, 4" instead of "crops" (as suggested by Curtis)

Reply:
I applied Curtis's recommendation and switched to using a `dict` of crops instead of a list. The boxes and scatter plot should all have relevant labels. Much improved!

b5e16e2aa65865d02794be0bb376ac20763397af